src/room.ts

```diff
@@ -25,12 +25,12 @@ type RoomObj = {
   topic:      string
   ownerUin:   number
   memberList: Contact[]
-  nickMap:    Map<string, string>
-  remarkMap:  Map<string, string>
-  displayMap: Map<string, string>
+  nameMap:      Map<string, string>
+  aliasMap:     Map<string, string>
+  roomAliasMap: Map<string, string>
```

Comment:
Remove `roomAliasMap` here. Because `aliasMap` should be `roomAliasMap`

The room should only in charge of search **alias in room**. Search contact alias should use `Contact.find()` instead.